README.md

```diff
@@ -47,3 +39,43 @@ The options include:
 
 Regardless of the workflow chosen, it is important that we strive to keep the git history clean.
 This means never force pushing on main, and usually having one commit per new post.
+
+## Install
+
+### Installation
+
+```
+$ yarn
+```
+
+### Local Development
+
+```
+$ yarn start
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+### Build
+
+```
+$ yarn build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+### Deployment
```

Comment:
Aren't the pages automatically deployed?

Reply:
That's the default README, I will remove